src/core/tests/type_prop/squeeze.cpp

```diff
@@ -261,7 +261,7 @@ TEST_P(SqueezeBoundTest, propagate_label_and_dynamic_value) {
     const auto unsqueeze = std::make_shared<op::v0::Unsqueeze>(gather, axis_1);
     const auto squeeze = std::make_shared<op::v0::Squeeze>(unsqueeze, axis);
 
-    const auto bc = std::make_shared<op::v1::Broadcast>(param, squeeze);
+    const auto bc = std::make_shared<op::v3::Broadcast>(param, squeeze);
```

Comment:
smth wrong with `v1::Broadcast` or just upgrading to use latest versions?

Reply:
Just upgrade to latest one, and @mitruska improve in this version some checks.